src/project-manager.ts

```diff
@@ -357,7 +357,7 @@ export class ProjectManager implements Disposable {
 				const info = ts.preProcessFile(contents, true, true);
 				const compilerOpt = config.getHost().getCompilationSettings();
 				for (const imp of info.importedFiles) {
-					const resolved = ts.resolveModuleName(imp.fileName, filePath, compilerOpt, config.moduleResolutionHost());
+					const resolved = ts.resolveModuleName(util.normalizePath(imp.fileName), filePath, compilerOpt, config.moduleResolutionHost());
```

Comment:
So `preProcessFile()` will return raw, relative, maybe-Windows imports, but `resolveModuleName()` only takes relative Posix paths?
Given the problem I commented further down, how about we first try to resolve the raw path, then if it didn't resolve, try again with backslashes replaced with forward slashes?